doc/requirements.txt

```diff
@@ -1,48 +0,0 @@
-plotly==6.3.0
-anywidget
-cufflinks==0.17.3
-dash-bio
```

Comment:
One of the tests is failing because this is missing. I think we can restore it for now and completely remove it once we take down the old docs